ios/keyman/Keyman/Keyman/AppDelegate.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  AppDelegate.swift
+//  Keyman
+//
+//  Created by Gabriel Wong on 2017-09-07.
+//  Copyright © 2017 SIL International. All rights reserved.
+//
+
+import UIKit
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+  private var _overlayWindow: UIWindow?
+
+  var window: UIWindow?
+  var viewController: MainViewController!
+  var navigationController: UINavigationController?
+
+  func application(_ application: UIApplication,
+                   didFinishLaunchingWithOptions launchOptions: [UIApplicationLaunchOptionsKey : Any]? = nil) -> Bool {
+    Thread.sleep(forTimeInterval: 1.0)
```

Comment:
BugReport: Why oh why do we sleep for a second on start? Is there any good reason for this?

Reply:
I believe it's to show off the beautiful splash screen.

Reply:
It needs to die.